DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "![](https://i.imgur.com/CoKDCuc.png)\n",
+    "\n",
+    "You'll (hopefully) not be surprised to know that there is no correlation in real life between cabbage and innie bellybuttons or tomatoes and Judaism. This is a function, as the article explains, of over 27,000 regressions being used over less than 60 complete responses, where they expect a 5% false-positive rate. Why 5%? well, that goes into the next topic, p-hacking.\n",
+    "\n",
+    "\n",
+    "#### P-Hacking and Cherry Picking\n",
+    "\n",
+    "You might have noticed the far-right column in the above picture has a \"p-value\" label. This is the value that statisticians use to measure the risk of a false positive. In general, studies with a p-value of less than 0.05, or 5%, can be published. It is a hard burden to prove in some situations, but in some situations it is absurdly easy. Linked in the above article is an [interactive piece](https://projects.fivethirtyeight.com/p-hacking/) by FiveThirtyEight that allows you to use p-hacking in action. This one comes with a bonus of letting you lean into confirmation bias as well, as it contains economic and political data from the last ~70 years. You get to cherry-pick the types of political data to use and the ways to measure a \"good\" economy, and the right panel allows you to see how significant your results are. It should go without saying that if you cherry-pick enough, you can prove any conclusion you want to.\n",
+    "\n",
+    "This phenomenon is very possible in the transfer of information at any stage from data sources to scientists to reporters to consumers, again intentional or unintentional. We see this taken to a comedic level with XKCD's exploration of jelly beans:\n",
+    "\n",
+    "![](https://i.imgur.com/k2dDf9e.png)\n",
+    "\n",
+    "\n",
+    "What you should take away here is that data manipulated is not meaningful, and I argue it is not even information, as it does not represent knowledge in any real way. Let's not be too cynical now, there _is_ meaningful information in the world, and we want to separate it from this darker part of the data sphere.\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Meaningful Information\n",
+    "\n",
+    "Here we have information, not misleading or malicious, and want to determine what to do with it. So then, what makes your information worth producing and using?\n",
+    "\n",
+    "#### \"Useful\" Information\n",
+    "\n",
+    "When we say \"useful\", what we want is SMART. SMART is an acronym that is generally used for goal-setting, where it stands for:\n",
+    "\n",
+    "- Specific: this information should be scoped to answer the question(s) you have, as information too broad risks being less applicable to the situation\n",
+    "- Measurable: an excellent metric for what to do with your data, and the subject of the next part\n",
```

Comment:
- Capitalize "an" at the beginning 
- The subject of the next part of what? Not sure if this segment is unfinished here.